internal/mapper/mapper.path_test.go

```diff
@@ -102,7 +103,8 @@ func Test_findPath_Cycle(t *testing.T) {
 
 func Test_GetPath_StartNotFound(t *testing.T) {
```

Comment:
Could convert to use assert like this 
```
func Test_GetPath_StartNotFound(t *testing.T) {
	_, err := GetPath(999999)

	assert.ErrorIs(t, err, ErrPathNotFound, "expected ErrPathNotFound for missing start")
}
```

Reply:
Done. Converted the whole file to use testify